ZFitter/bin/ZFitter.cpp

```diff
@@ -160,13 +160,17 @@ void UpdateFriends(tag_chain_map_t& tagChainMap, TString regionsFileNameTag)
 				if(chain->GetFriend(chain_itr->first) == NULL) {
 					std::cout << "[STATUS] Adding friend branch: " << chain_itr->first
 					          << " to tag " << tag_chain_itr->first << std::endl;
-					chain->AddFriend(chain_itr->second.get());
+					  chain->AddFriend(chain_itr->second.get());
 				} // already added
 			}
 
-			if(chain->GetEntries() != chain_itr->second->GetEntries()) {
-				std::cerr << "[ERROR] Not the same number of events: " << chain->GetEntries() << "\t" << chain_itr->second->GetEntries() << std::endl;
-				exit(1);
+			if(chain->GetEntries() != chain_itr->second->GetEntries() && chain_itr->first!="scaleEle") {//Not scaleEle: check data and MC
```

Comment:
Normally addBranch creates friend root trees for data and MC and then checks the number of events. I want this check to be done only on data if the friend branch is "scaleEle" (no scaleEle branch for MC)